### DISCUSSION TITLE
Position optimization using DAFoam

### DISCUSSION THREAD

**Discussion by uttamcadambi07** [mdolab/dafoam#394] [General]
Hi,

Is it possible to do a position optimization using DAFoam?

For example - I have a cylinder and have some reference field. I define a correlation function which tells me how close my baseline data is to the reference case (like an objective function). I would like to have a design variable x,y,z,theta which helps me move my cylinder in x,y,z directions to increase the correlation. I assume I can begin this with looking at FFD points since that does shape optimization. Is it possible to extend it to positional changes with the available tools?

**Comment by friedenhe** [comment]
Yes, the FFD points can do translational change. For example, if you move all the FFD points in the x direction by 1 m, it will translate the design surface in the x direction by about 1 m.
